deployment/construct.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.0.2
```

Comment:
It would be nice to not hard-code this, but instead get it from here: https://github.com/computational-cell-analytics/micro-sam/blob/master/micro_sam/__version__.py, which is supposed to be "the single source of truth" regarding the version string. (Or alternatively you could also use the github tag, which should always be the same).

I am not sure about the best strategy for this, maybe a python script that takes this file as a template and then changes it to the correct version and resaves.